Package.swift

```diff
@@ -12,17 +12,64 @@ let package = Package(
         .package(url: "https://github.com/apple/swift-nio.git", from: "2.17.0")
     ],
     targets: [
-        .target(name: "Baggage"),
-        .testTarget(name: "BaggageTests", dependencies: ["Baggage"]),
+        // ==== --------------------------------------------------------------------------------------------------------
+        // MARK: Baggage
 
-        .target(name: "Instrumentation", dependencies: ["Baggage"]),
-        .testTarget(name: "InstrumentationTests", dependencies: ["Instrumentation", "Baggage"]),
+        .target(
+            name: "Baggage",
+            dependencies: []
+        ),
+        .testTarget(
+            name: "BaggageTests",
+            dependencies: [
+                "Baggage"
+            ]
+        ),
 
-        .target(name: "NIOInstrumentation", dependencies: [
-            .product(name: "NIO", package: "swift-nio"),
-            .product(name: "NIOHTTP1", package: "swift-nio"),
-            "Instrumentation"
-        ]),
-        .testTarget(name: "NIOInstrumentationTests", dependencies: ["NIOInstrumentation", "Instrumentation"])
+        // ==== --------------------------------------------------------------------------------------------------------
```

Comment:
Love it. That's a lot more readable now 👍

Reply:
Cool, yeah sorry for imposing my usual style here :) Glad you liked it.